### PR TITLE
add get_jupyter_token()

### PR DESCRIPTION
**Pull request by forana**

### CODE REVIEW

File: saturn_client/core.py [hunk 1]
```diff
@@ -305,6 +305,21 @@ def get_jupyter_server(self, jupyter_server_id) -> Dict[str, Any]:
             ) from err
         return response.json()
 
+    def get_jupyter_token(self, jupyter_server_id) -> str:
```
Comment (jameslamb) [inline saturn_client/core.py:308]:
```suggestion
    def get_jupyter_token(self, jupyter_server_id: str) -> str:
```

**Review by jameslamb — CHANGES_REQUESTED** [review]
What would this be used for that you can't do with your regular user API token?

File: saturn_client/core.py (hunk 1 above)
Comment (forana) [reply saturn_client/core.py:308]: I copy-pasted from the _one_ method that was missing this :upside_down_face:

**Comment by forana** [comment]
Closing - this functionality turned out to not be needed

**Pull request closed without merging**